pysumma/pysumma/Plotting.py

```diff
@@ -0,0 +1,111 @@
+import seaborn as sns
+import xarray as xr
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import pandas as pd
+
+class Plotting:
```

Comment:
In all of the `plot_xxx` methods they take a `ds` argument, which means that none of them require any class infrastructure.